### DISCUSSION TITLE
大ㄐㄐ

### DISCUSSION THREAD

**Discussion by sdsd122** [share121/inter-knot#1191] [其他]
我最近在空洞內遇到了一些奇怪的以骸，他們似乎是由兩個圓形物體跟一個...痾 圓柱體?
我不知道 有人些叫他阿姆什麼的 記不太清楚了 反正在線徵求繩降幫忙消滅那些以骸!

**Comment by sdsd122** [comment]
不是 啊？
我只是测试啊啊啊啊啊！ ！ ！ ！ ！ ！ ！ ！
我不小心按到发布了！ ！ ！ ！ ！ ！ ！ ！
让我死吧
![螢幕擷取畫面 2023-07-02 184800](https://github.com/user-attachments/assets/ecabecc1-171d-4eda-a99d-f6976972bb00)

**Comment by int-maxxx** [comment]
哈哈哈

**Comment by sky-R6** [comment]
難道是阿姆斯特朗回旋加速噴氣式阿姆斯特朗炮？
我聽說這種以骸很難纏

**Comment by Myitian** [comment]
标题和内容都可以编辑的，觉得没写好可以改的
